pyproject.toml

```diff
@@ -19,11 +19,16 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-test = ["pytest"]
+test = [
+    "nipoppy[bidscoin]",
+    "pytest"
+]
 
 # alias in case of typo
 tests = ["nipoppy[test]"]
 
+bidscoin = ["bidscoin>3.7.1"]
```

Comment:
I'm not sure I would add BIDSCOIN as a dependency (optional or not) since the goal would be for people to use the container (and have automatic pulling of the container if none is given).